module/dkg/client.go

```diff
@@ -256,7 +256,7 @@ func (c *Client) SubmitResult(groupPublicKey crypto.PublicKey, publicKeys []cryp
 
 // trim0x trims the `0x` if it exists from a hexadecimal string
 // This method is required as the DKG contract expects key lengths of 192 bytes
-// the `PublicKey.String()` method returns the hexadecimal string representation of the
+// the `PublicKey.Name()` method returns the hexadecimal string representation of the
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig